poh/src/poh_recorder.rs

```diff
@@ -263,6 +261,7 @@ impl PohRecorder {
                 tick_cache: vec![],
                 working_bank: None,
                 working_bank_sender,
+                shared_working_bank: SharedWorkingBank::default(),
```

Comment:
nit: I'd rather have this be SharedWorkingBank::empty() or something

IMO default is an anti pattern when the default isn't obvious, and in this case
it isn't clear what a default SharedWorkingBank is

Reply:
7c4ffe7e6eb1a499ed5a5eae6043e59b0cee5a30